src/butr.js

```diff
@@ -486,7 +488,13 @@ export const marker = options => {
    * @param {Node} activeLink currently active link
    */
   const setMarkerPosition = activeLink => {
-    marker.style.transform = `translateY(${activeLink.offsetTop}px)`
+    let translatePos = activeLink.offsetTop
+    const style = window.getComputedStyle(activeLink)
+    if (style.getPropertyValue('box-sizing') === 'border-box') {
+      translatePos -= Math.round(parseFloat(style.getPropertyValue('border-top-width'), 10))
+    }
+    marker.style.transform = `translateY(${translatePos}px)`
+    marker.style.height = `${activeLink.offsetHeight}px`
   }
```

Comment:
👍

Reply:
Didn't know you would need to round a border width that was _returned_ by the damn browser...